src/components/index.js

```diff
@@ -1,14 +1,21 @@
+/* eslint-disable no-use-before-define */
+/* eslint-disable no-plusplus */
+/* eslint-disable react/no-this-in-sfc */
+/* eslint-disable no-extend-native */
+/* eslint-disable import/no-extraneous-dependencies */
 import React, { useState, useEffect, useRef } from 'react';
 import XLSX from 'xlsx';
-import { Button, Modal, Form, Input } from 'antd';
+import {
+  Button, Modal, Form, Input,
+} from 'antd';
```

Comment:
Please make  this import statement a one-liner to make it consistent with the rest

Reply:
Done